src/content/docs/it/tutorial/0-introduction/1.mdx

```diff
@@ -0,0 +1,59 @@
+---
+type: tutorial
+title: A proposito di questo tutorial
+description: Tutto quello che ti serve per iniziare con il tutorial "Costruisci il tuo primo blog con Astro".
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+
+
+## Cosa mi serve prima di iniziare?
+
+Se hai già un po' di dimestichezza con **HTML**, **Markdown**, **CSS** e un po' di **JavaScript**, allora sei pronto per cominciare. Potrai portare a termine il tutorial semplicemente seguendo le istruzioni. Astro è per tutti! 🧑‍🚀 👩‍🚀 👨‍🚀
+
+Inoltre avrai bisogno di un account [GitHub](https://github.com) (o qualcosa di analogo) per pubblicare il tuo progetto sul web. 
+
+<details>
+<summary>A cosa serve la checklist in fondo ad ogni pagina?</summary>
+
+In fondo ad ogni pagina troverai una checklist di attività che dovresti essere in grado di portare a termine. Spunta le voci per vedere il tuo progresso nel Tutorial Tracker.
+
+L'utilizzo del Tracker è opzionale, ma ti aiuterà a ricordare dove sei arrivato, nel caso in cui decidi di completare il tutorial in tempi diversi. Puoi anche lasciare alcune checklist incomplete, come promemoria di sezioni che vuoi rivedere in un secondo momento.
+
+(Queste informazioni sono salvate esclusivamente nel local storage del tuo browser. Nessuna informazione viene inviata, né viene salvata, da Astro.)
```

Comment:
This might sound a nitpicky suggestion, but that passive form is not completely correct in Italian.

```suggestion
(Queste informazioni sono salvate esclusivamente in locale sul tuo browser. Astro non raccoglie né salva alcuna informazione.)
```